### ISSUE TITLE
distal radioulnar joint (DRUJ) translation

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10747]
**Preferred term label:**
distal radioulnar joint (DRUJ) translation

**Synonyms**

**Definition (free text, please give PubMed ID)**
CT scan showing subluxation of the distal ulna either volar, dorsal, or lateral such that it is no longer congruent in the sigmoid notch

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Distal radioulnar joint (DRUJ) instability
Distal radius malunion

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:5000049
- label: Distal radioulnar joint translation
- parents: [Abnormality of the radioulnar joints (HP:0003059)](https://hpo.jax.org/app/browse/term/HP:0003059)
- pmid: PMID:36399415
- def: Abnormal linear movement or displacement of the ulnar head relative to the radius at the wrist, indicating instability of the distal radioulnar joint, which is the distal articulation between the radius and ulna, acting as a major weight-bearing joint at the wrist and distributing forces across the forearm bones.
- comment: CT scan shows subluxation of the distal ulna either volar, dorsal, or lateral such that it is no longer congruent in the sigmoid notch.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10747
- synonyms:
